Blockzilla/AppDelegate.swift

```diff
@@ -278,6 +278,15 @@ extension AppDelegate {
             telemetryConfig.isUploadEnabled = Settings.getToggle(.sendAnonymousUsageData)
         #endif
         
+        Telemetry.default.beforeSerializePing(pingType: CorePingBuilder.PingType) { (inputDict) -> [String : Any?] in
+            var outputDict = inputDict // make a mutable copy
+            
+            let shouldShowTrackerStatsToUser = UserDefaults.standard.object(forKey: BrowserViewController.userDefaultsShareTrackerStatsKey) as! Bool?
+            outputDict["showTrackerStatsShare"] = shouldShowTrackerStatsToUser
```

Comment:
We'll want to make sure `shouldShowTrackerStatsToUser` is a Boolean, not an Optional. (I don't think Telemetry unwraps things for us, just uses the description when serializing to JSON.

This could work
```
let shouldShowTrackerStatsToUser = (UserDefaults.standard.object(forKey: BrowserViewController.userDefaultsShareTrackerStatsKey) as? Bool) ?? false
```

Reply:
Would that report back as False even if the value is nil? Should we just not include the param if it is nil(meaning the coin flip hasn’t been done)

Reply:
You know what? We have a reference to `BrowserViewController`. We could just do

```
outputDict["showTrackerStatsShare"] = browserViewController.shouldShowTrackerStatsShareButton()
```

I'm kinda okay with making that public for now since it will encourage us to clean it up if we decide to turn this on for all users.

Reply:
great! made that change.